sycl/test-e2e/Basic/cospi.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %{build} -o %t.out
```

Comment:
Please move the test to https://github.com/intel/llvm/tree/6ba140044371a85af97e051fb19b53d948721992/sycl/test-e2e/Basic/built-ins , this is a folder for all builtins tests. (and we have a legacy folder for built-ins - https://github.com/intel/llvm/tree/6ba140044371a85af97e051fb19b53d948721992/sycl/test-e2e/DeviceLib/built-ins , these tests were written before SYCL 2020, and we will someday move it to test-e2e/Basic/built-ins)

Reply:
BTW, can the test be renamed to something like `host_cospi.cpp`? Or to `host_math.cpp` if other math builtins will be added to this source file.